### DISCUSSION TITLE
How to add the MDX Editor drawer?

### DISCUSSION THREAD

**Discussion by jackschd** [atoti/atoti#498] [Q&A] [answered]
Can we customise the ActiveUI settings from within the atoti python API?  
Specifically, how can we add the MDX editor to the drawer which isn't available by default but is available as an extension?

**Comment by tibdex** [comment, marked as answer]
In the upcoming release, it will be possible to enable the MDX editor and other advanced UI features like this:

```diff
  import atoti as tt
+ from atoti_plus import ADVANCED_APP_EXTENSION
- session = tt.Session()
+ session = tt.Session(app_extensions=dict([ADVANCED_APP_EXTENSION])) 
```

This is an Atoti+ only feature.

**Reply by fabiencelier** [reply]
Awesome ! Thank you !

**Reply by jackschd** [reply]
Excellent, thanks!
